### DISCUSSION TITLE
show query string

### DISCUSSION THREAD

**Discussion by dnagirl** [Innofactor/XrmToolBox.BulkDataUpdater#154] [Ideas]
One of the features I love about FetchXML is that I can see the query in multiple formats-- particularly OData 4.0. It would be wonderful if BulkDataUpdater had the option to display the query it was about to run.

**Comment by rappen** [comment]
Trying to understand 🙂🤔

You want to update bulk data, and then see the OData.
Why?
Are you using BDU instead of FXB, and want to "get the code" for the queries?
Or you want to write the OData to query the data?

**Reply by dnagirl** [reply]
> Are you using BDU instead of FXB, and want to "get the code" for the queries?

yes, exactly. Documentation for OData is not very educational. I find it easier to learn the syntax when I can see the OData  version of an operation I'm about to run.

**Reply by rappen** [reply]
I see (I think so).
So read OData is easier than FetchXML, right?
Would be even better to read the filter like this in *Advanced filters*, something like this (just an example)?

![image](https://user-images.githubusercontent.com/2572253/156737622-a7c73ee7-2984-44aa-956d-ed9708eb7586.png)

Easier to read? 🙂
